node/src/debian/DEBIAN/postinst.sh

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env bash
```

Comment:
This file should be renamed to `postinst` (with **no** .sh suffix).

Currently, as named, it is not actually running.  The reason you still see an `rnode` user + group being created is that is part of the default behavior of `sbt-native-packager` (though only for debian), but I'd prefer to do it ourselves explicitly in this script.